Packs/Zafran/Integrations/Zafran/Zafran.yml

```
@@ -0,0 +1,136 @@
+commonfields:
+  id: Zafran API
+  version: -1
+name: Zafran API
+display: Zafran API
+category: Utilities
+description: 'The Zafran API provides a programmatic way to interact with the Zafran Risk & Mitigation Platform for various use cases.  Using the API requires an   API token  , which can be obtained by contacting the Zafran support team at support@zafran.io.  API authorization is based on a bearer token that should be added as a header to all requests made for the AP, in the following way:  `Authorization: Bearer  token ` '
+configuration:
+- name: url
+  display: Server URL (e.g. api.zafran.io)
+  defaultvalue: https://api.zafran.io # disable-secrets-detection
```

```suggestion
  defaultvalue: https://api.zafran.io
```

add `https://api.zafran.io` to **.secrets-ignore** file